scripts/install_third_party.py

```diff
@@ -380,6 +380,12 @@ def download_and_untar_files(
 NG_IMG_CROP_ZIP_ROOT_NAME = 'ngImgCrop-%s' % NG_IMG_CROP_REV
 NG_IMG_CROP_TARGET_ROOT_NAME = 'ng-img-crop-%s' % NG_IMG_CROP_REV
 
+NODE_JSCS_REV = '2.3.0'
+NODE_JSCS_ZIP_URL = (
+    'https://github.com/jscs-dev/node-jscs/archive/v%s.zip' %NODE_JSCS_REV)
```

Comment:
Actually -- can this just be installed in the bash scripts using node, similar to what we do for e.g. karma-coverage etc.?